xbmc/addons/kodi-addon-dev-kit/Makefile

```diff
@@ -0,0 +1,30 @@
+ifeq ($(CMAKE),)
+  CMAKE = cmake
+endif
+
+ifeq ($(PREFIX),)
+  PREFIX = /usr/local
+endif
+
+all: .create-kit
+
+.cmake-kit:
```

Comment:
this target and create-kit should probably be .phony
cmake will take care to not uselessly rebuild stuff.

and deleting the build dir shouldn't be necessary